Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -351,6 +351,7 @@ public override void Dispose()
                 emojiSuggestionPanelController.OnEmojiSelected -= AddEmojiFromSuggestion;
 
             dclInput.UI.Submit.performed -= OnSubmitAction;
+            emojiPanelController.Dispose();
```

Comment:
Is this a left over?
`emojiPanelController.Dispose();` is being called in line 347 after a good null check.

Reply:
good catch, I think it was something related to a merge conflict and I missed it, fixed!